examples/wasmer.sh/index.ts

```diff
@@ -15,7 +17,7 @@ async function main() {
     // See https://github.com/wasmerio/wasmer-js/issues/373
     const { Wasmer, init, initializeLogger } = await import("@wasmer/sdk");
 
-    await init();
+    await init(wasmerSDKUrl);
```

Comment:
Will all users of `@wasmer/sdk` need to do this if they are importing the non-embedded build? 

If so, I'd prefer not to merge this PR without changing `init()` so you can call it without needing to worry about `wasmerSDKUrl` or any of that. It's important for DX that initializing the package *Just Works* out of the box.

Reply:
No, this is not needed. We use it so we can use a local wasm file instead of the unpkg one. But this one is not required